makefiles/vars.inc.mk

```diff
@@ -28,6 +28,7 @@ export RIOTPROJECT           # Top level git root of the project being built, or
 export RIOTMAKE              # Location of all supplemental Makefiles (such as this file)
 export BINDIRBASE            # This is the folder where the application should be built in. For each BOARD a different subfolder is used.
 export BINDIR                # This is the folder where the application should be built in.
+export BUILDOUT_DIR          # The base folder for output of the RIOT build process
```

Comment:
Some subjective feedback.

I disagree with the current documentation and then the name of the variable.
What means 'output' here ? (which leads to buildOUT_dir).

If this folder is now supposed to store other things than test results it should have a more generic documentation and name.
I see it more as a folder used for "common build things", also with a subdirectory for "build artifacts(results/outputs)".
Maybe simply "BUILDDIR" could do, with a documentation saying it is a "common" one to differentiate to `BINDIR*` that are `application` specific. But I am bad at naming…

Reply:
IIRC the naming *buildout* came up during our f2f discussion a while ago at the HAW. But I'm fine with changing the name and documentation.